ApiEverest/CrudApi/Controllers/CostumerController.cs

```diff
@@ -43,7 +47,7 @@ public IActionResult Create([FromBody] CustomerEntity customerEntity)
         {
             try
             {
-                _customerService.Create(customerEntity);
+                _customerAppService.Create(customerEntity);
                 return Created("Id:", customerEntity.Id);
```

Comment:
```suggestion
                return Created("Id:", customerId");
```
Utilizar o retorno do método Create de _customerAppService